runbotics/runbotics-orchestrator-ui/src/src-app/Actions/excel.actions.ts

```diff
@@ -179,6 +179,105 @@ const getExcelActions: () => Record<string, IBpmnAction> = () => ({
             formData: {},
         },
     },
+    [ExcelAction.CREATE_WORKSHEET]: {
+        id: ExcelAction.CREATE_WORKSHEET,
+        label: translate('Process.Details.Modeler.Actions.Excel.CreateWorksheet.Label'),
+        script: ExcelAction.CREATE_WORKSHEET,
+        runner: Runner.DESKTOP_SCRIPT,
+        system: ActionSystem.WINDOWS,
+        output: {
+            assignVariables: true,
+            outputMethods: {
+                variableName: '${content.output[0]}',
+            },
+        },
+        form: {
+            schema: {
+                type: 'object',
+                properties: {
+                    input: {
+                        title: translate('Process.Details.Modeler.Actions.Common.Input'),
+                        type: 'object',
+                        properties: {
+                            name: {
+                                title: translate('Process.Details.Modeler.Actions.Excel.CreateWorksheet.Name'),
+                                type: 'string'
+                            },
+                            activate: {
+                                title: translate('Process.Details.Modeler.Actions.Excel.CreateWorksheet.Activate'),
+                                type: 'boolean',
+                                default: true,
+                            },
+                        },
+                    },
+                    output: {
+                        title: translate('Process.Details.Modeler.Actions.Common.Output'),
+                        type: 'object',
+                        properties: {
+                            variableName: {
+                                title: translate('Process.Details.Modeler.Actions.Common.VariableName'),
+                                description: translate('Process.Details.Modeler.Actions.Common.VariableMessage'),
+                                type: 'string',
+                            },
+                        },
+                        required: ['variableName'],
```

Comment:
do we neccessesarily want to use returned worksheet? why do we even return created worksheet for?

Reply:
cause if some1 doesn't give a name, the name Excel gives to the new worksheet is returned and can be further used